main.go

```diff
@@ -74,6 +74,8 @@ func main() {
 		"If set the metrics endpoint is served securely")
 	flag.BoolVar(&conf.EnableHTTP2, "enable-http2", false,
 		"If set, HTTP/2 will be enabled for the metrics and webhook servers")
+	flag.StringVar(&conf.Namespace, "namespace", "",
+		"The namespace where the observability-operator is running.")
```

Comment:
Do we care about the namespace the observability operator runs in or where we expect the alertmanager secret to be? Because if it's the latter, I would rename this to alertmanager-secret-namespace and move it to the desired location

Reply:
The secret is created in the same namespace as the operator, this namespace is taken from the helm chart value which defined the namespace for the entire chart, therefore I named it namespace to reflect this.

Reply:
But that would be incorrect that's not only the alertmanager secret's namespace

Reply:
It's not but it's only used for that right now right?

Reply:
What is your reasoning behind it?

And if you do not want to name it alertmanager-secret-namespace then can we at least name it operator-namespace or something? Because namespace means nothing tbh